addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/internal/handler/FreeboxHandler.java

```diff
@@ -215,161 +148,128 @@ public void initialize() {
+            errorMsg = fqdn + " does not deliver any API version";
+        } else if (Boolean.TRUE.equals(result.isHttpsAvailable()) && !Boolean.TRUE.equals(configuration.useOnlyHttp)) {
+            if (result.getHttpsPort() == null || StringUtils.isEmpty(result.getApiDomain())) {
+                if (httpsRequestOk) {
                     useHttps = true;
-                    fqdn = apiDomain + ":" + httpsPort;
+                } else {
+                    logger.debug("{} does not deliver API domain or HTTPS port; use HTTP API", fqdn);
                 }
+            } else if (apiManager.checkApi(String.format("%s:%d", result.getApiDomain(), result.getHttpsPort()),
+                    true) != null) {
+                useHttps = true;
+                fqdn = String.format("%s:%d", result.getApiDomain(), result.getHttpsPort());
             }
         }
 
         if (errorMsg != null) {
             logger.debug("Thing {}: bad configuration: {}", getThing().getUID(), errorMsg);
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, errorMsg);
-        } else if (!authorize(useHttps, fqdn, apiBaseUrl, apiVersion)) {
+        } else if (!apiManager.authorize(useHttps, fqdn, result.getApiBaseUrl(), result.getApiVersion(),
+                configuration.appToken)) {
             if (StringUtils.isEmpty(configuration.appToken)) {
                 errorMsg = "App token not set in the thing configuration";
             } else {
                 errorMsg = "Check your app token in the thing configuration; opening session with " + fqdn + " using "
-                        + (useHttps ? "HTTPS" : "HTTP") + " API version " + apiVersion + " failed";
+                        + (useHttps ? "HTTPS" : "HTTP") + " API version " + result.getApiVersion() + " failed";
             }
             logger.debug("Thing {}: {}", getThing().getUID(), errorMsg);
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, errorMsg);
         } else {
             logger.info("Thing {}: session opened with {} using {} API version {}", getThing().getUID(), fqdn,
-                    (useHttps ? "HTTPS" : "HTTP"), apiVersion);
-            updateStatus(ThingStatus.ONLINE);
-
+                    (useHttps ? "HTTPS" : "HTTP"), result.getApiVersion());
             if (globalJob == null || globalJob.isCancelled()) {
                 long pollingInterval = getConfigAs(FreeboxServerConfiguration.class).refreshInterval;
                 logger.debug("Scheduling server state update every {} seconds...", pollingInterval);
-                globalJob = scheduler.scheduleWithFixedDelay(globalRunnable, 1, pollingInterval, TimeUnit.SECONDS);
+                globalJob = scheduler.scheduleWithFixedDelay(() -> {
+                    try {
+                        pollServerState();
+                    } catch (Exception e) {
+                        logger.info("Server state job failed: {}", e.getMessage(), e);
```

Comment:
Reduce logging level.

Reply:
Done

Reply:
Done but I am not happy with this change as any potential problem will now occur fully silently with nothing to inform the user in the logs.